catalog-rest-service/src/main/java/org/openmetadata/catalog/exception/CatalogGenericExceptionMapper.java

```diff
@@ -42,6 +42,7 @@ public CatalogGenericExceptionMapper() {}
 
   @Override
   public Response toResponse(Throwable ex) {
+    ex.printStackTrace();
```

Comment:
Turn this into log or do we intend to return full stack trace here?

Reply:
Deleted the line used for debugging